src/mintpy/objects/gps.py

```diff
@@ -508,10 +509,30 @@ def get_los_geometry(self, geom_obj, print_msg=False):
             az_angle  = ut.heading2azimuth_angle(float(geom_obj['HEADING']))
 
         else:
-            raise ValueError(f'input geom_obj is neight str nor dict: {geom_obj}')
+            raise ValueError(f'input geom_obj is neither str nor dict: {geom_obj}')
 
-        return inc_angle, az_angle
+        return inc_angle or np.nan, az_angle or np.nan
```

Comment:
I am just curious: in this syntax, when will the `np.nan` value be returned? Is it the same scenario as the `raise ValueError` above?